CIT365_W9_MegaDeskV2/Pages/DeskQuotes/Create.cshtml.cs

```diff
@@ -8,39 +8,43 @@
 using CIT365_W9_MegaDeskV2.Data;
 using CIT365_W9_MegaDeskV2.Models;
 using System.Web;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
 
 namespace CIT365_W9_MegaDeskV2.Pages.DeskQuotes
 {
-    public class CreateModel : PageModel
+    public class CreateModel : DeskQuotePageModel
     {
         private readonly CIT365_W9_MegaDeskV2.Data.CIT365_W9_MegaDeskV2Context _context;
+        private readonly IConfiguration _configuration;
 
         public List<SelectListItem> Countries { get; } = new List<SelectListItem>
```

Comment:
This code was accidentally left in by me.

Reply:
Hey Dan! I'm actually using the configuration to pull in the pricing data, see https://github.com/ajlane17/cit365-group/pull/3/files#diff-d35cf59d0706e7805bc0a3daf5616e27 for details